### PR TITLE
[POAE7-2362] update develop-guide.html

### PR DESCRIPTION
**Pull request by imajiayu**
### What changes were proposed in this pull request?
Modified /docs/develop-guide.rts to update https://intel.github.io/BDTK/develop-guide.html
mainly update how to build BDTK.

### Why are the changes needed?
The github repo in old guidance have been discarded.  

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
no need

### CODE REVIEW

**Comment by jikunshang** [comment]
can you check why `syntax-support.rst` is not shown in [web page](https://intel.github.io/BDTK/)?

File: docs/develop-guide.rst [hunk 1]
```diff
@@ -71,17 +60,13 @@ details are as follows:*
 
 1. Build in container
 
-::
+3. Build in container
```
Comment (xieqi) [inline docs/develop-guide.rst:63]: repeat with L61

File: docs/develop-guide.rst [hunk 2]
```diff
@@ -71,17 +60,13 @@ details are as follows:*
...
-   # default target is release version
-   make debug/release
+   Once you have setup the Docker build envirenment for BDTK and get the source, you can enter the BDTK container and build like:
 
-   # incremental compilation method:
-   # 1. continue use 'make debug/release', some cmake actions will spend a little time
-   # 2. use 'make build BUILD_TYPE=Debug/Release', the fastest method
+   Run ``make`` in the root directory to compile the sources. For development, use ``make debug`` to build a non-optimized debug version, or ``make release`` to build an optimized version. Use ``make test-debug`` or ``make test-release`` to run tests.
 
+   Run ``make`` in the root directory to compile the sources. For development, use ``make debug`` to build a non-optimized debug version, or ``make release`` to build an optimized version. Use ``make test-debug`` or ``make test-release`` to run tests.
```
Comment (xieqi) [inline docs/develop-guide.rst:69]: repeat with L67

File: docs/develop-guide.rst [hunk 3]
```diff
@@ -118,10 +103,9 @@ For velox-plugin:
 
 
 2.2 How to run a single unit test
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
 
 ::
+   
    # For Cider
    # Test Path: /workspace/velox-plugin/build-$BUILD_TYPE/cider/Tests
```
Comment (xieqi) [inline docs/develop-guide.rst:110]: Tests --> tests

File: docs/develop-guide.rst [hunk 4]
```diff
@@ -4,44 +4,33 @@ Cider Developer Guide
...
-   #    - config 'NTP' /etc/systemd/timesyncd.conf, you need to find a service in your company or elsewhere.
-   #    - run 'systemctl restart systemd-timesyncd.service'
-   # 2. add parameters after apt-get in Dockerfile to skip timezone check, like:
-   #    RUN apt-get -o Acquire::Check-Valid-Until=false -o Acquire::Check-Date=false update 
+   # Build an image from a Dockerfile
+   $ cd ${path_to_source_of_bdtk}/ci/docker
+   $ docker build -t ${image_name} .
 
    # Start a docker container for development
    docker run -d --name ${container_name} --privileged=true -v ${path_to_velox_plugin}:/workspace/velox-plugin ${image_name} /usr/sbin/init
```
Comment (xieqi) [inline docs/develop-guide.rst:36]: velox-plugin  --> BDTK

File: docs/develop-guide.rst [hunk 5]
```diff
@@ -132,8 +116,8 @@ For velox-plugin:
    # Please check is there a exec directory(auto generate when make test-$BUILD_TYPE) under cider-velox.
```
Comment (xieqi) [inline docs/develop-guide.rst:116]: exec --> function

File: docs/develop-guide.rst (hunk 4 above)
Comment (imajiayu) [reply docs/develop-guide.rst:36]: do I need to change all the "velox-plugin" to BDTK?

File: docs/develop-guide.rst [hunk 6]
```diff
@@ -132,8 +116,8 @@ For velox-plugin:
...
    # If not, please use follow commands to get one(under /workspace/velox-plugin/): 
    #   mkdir /workspace/velox-plugin/build-${BUILD_TYPE}/cider-velox/function
-	#   cp /workspace/velox-plugin/build-${BUILD_TYPE}/cider/function/*.bc /workspace/velox-plugin/build-{BUILD_TYPE}/cider-velox/function
-	#   cd /workspace/velox-plugin/build-${BUILD_TYPE}cider-velox/test
+   #   cp /workspace/velox-plugin/build-${BUILD_TYPE}/cider/function/*.bc /workspace/velox-plugin/build-{BUILD_TYPE}/cider-velox/function
```
Comment (winningsix) [inline docs/develop-guide.rst:119]: Hmm, did you try to this locally? We should all use BDTK instead now.

**Review by winningsix — APPROVED** [review]

**Review by kyotoyx — APPROVED** [review]

**Review by winningsix — APPROVED** [review]

**Review by xieqi — APPROVED** [review]

**Pull request merged**